xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -988,6 +988,13 @@ CEpgInfoTagPtr CPVRTimerInfoTag::GetEpgInfoTag(bool bCreate /* = true */) const
   return m_epgTag;
 }
 
+void CPVRTimerInfoTag::SetEpgTag(const CEpgInfoTagPtr &tag)
+{
+  CSingleLock lock(m_critSection);
```

Comment:
Is this really an issue? SetEpgTag is not called that often (once per timer per timer list update) and is not performance critical at all.